slapp/data_selection/README.rst

```diff
@@ -0,0 +1,115 @@
+
+  In [6]: print(base64.b64decode(results['base64_query_strings'][0]))
+
+  b"SELECT oe.id as exp_id, idep.depth, genotypes.name FROM ophys_experiments as oe JOIN ophys_sessions as os on oe.ophys_session_id=os.id JOIN projects on projects.id=os.project_id JOIN imaging_depths as idep on idep.id = oe.imaging_depth_id JOIN specimens on os.specimen_id=specimens.id JOIN donors on specimens.donor_id=donors.id JOIN donors_genotypes dg ON dg.donor_id=donors.id JOIN genotypes ON genotypes.id=dg.genotype_id JOIN genotype_types as gt on gt.id=genotypes.genotype_type_id WHERE gt.name='driver' AND oe.workflow_state = 'passed' AND projects.code='VisualBehaviorMultiscope' AND genotypes.name='Sst-IRES-Cre' AND idep.depth BETWEEN 100 and 300"
+  
+The field ``sub_selected_ids`` is intended to be a list of LIMS ``ophys_experiments.id`` that will be sent through segmentation.
+
+::
+
+  In [7]: print(repr(results['sub_selected_ids']))                                   
+  [851093291, 977247468, 866518324, 853363749, 865798237, 951980481, 982903847, 986518870, 977978331, 986518863, 982903843, 957759564, 978296102, 988707128, 960995086, 867410514, 988707124, 867410520, 978296114, 953659743, 977247476, 867410518, 960995077, 982903853, 977978329, 853363739, 987317107, 856123119, 856123117, 953659752, 982344777, 853988446, 871196369, 976300303, 866518318, 871196375, 957759568, 866518326, 960995084, 868870094, 871196377, 850517344, 986518852, 866518314, 957759562, 956941841, 959388794, 989212489, 959388790, 976300297, 987317101, 864967106, 866518316, 977978327, 951980479, 854759894, 866518293, 871196365, 850517352, 875786885, 853988430, 864430668, 956941848, 868870092, 857698006, 868870085, 959388798, 977247474, 865798247, 958527479, 864967102, 988707126, 977247472, 873963899, 864430666, 994053903, 1002314807, 989191384, 1003771249, 1010812025, 986402309, 979668410, 960960480, 993344860, 957652800, 1001535125, 978827848, 993862620, 994061182, 1003456269, 1012112426, 995439942, 994790561, 984551228, 993593393, 919419001, 974433390, 889806727, 905955238, 889806719, 908381674, 932381896, 908381700, 914107592, 990681006, 886585126, 974433399, 909184300, 914580660, 989610989, 915243090, 972233193, 916220450, 905955228, 929603796, 887386949, 904363934, 886003523, 989610985, 929603805, 901559828, 935440149, 971761068, 934456506, 1011751579, 906877227, 932333410, 972683314, 994278291, 915141818]
+
+Create Segmentation Manifest
```

Comment:
I feel like this maybe belongs more in ophys_segmentation, but it's fine to keep here.

Reply:
Leaving it for now.